packages/appstore/src/components/trade-button/trade-button.tsx

```diff
@@ -1,9 +1,9 @@
 import { Button } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
 import { localize } from '@deriv/translations';
 import { Actions } from 'Components/containers/trading-app-card-actions';
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { useStores } from 'Stores';
 
 const TradeButton = ({
     link_to,
```

Comment:
since we are using observer from usestore, we can wrap observer as below
const TradeButton = observer({
    link_to,...)}


    export default TradeButton